ros/src/tl_detector/tl_detector.py

```diff
@@ -101,7 +107,15 @@ def get_closest_waypoint(self, pose):
 
         """
         #TODO implement
-        return 0
+        # print "type(pose)=%s" % (type(pose))
+        # print "type(pose.position)=%s" % (type(pose.position))
+
+        # print [pose.position.x, pose.position.y]
+        # point = [pose.position.x, pose.position.y]
+        # print point
+        # print x, y
+        
+        return self.waypoint_tree.query([x, y])[1]
```

Comment:
In some of the other python scripts, we added a ([x,y],1)[1] i.s.o. ([x,y])[1] for the tree query. [KDTree Documentation](https://docs.scipy.org/doc/scipy/reference/generated/scipy.spatial.KDTree.html) says, this is telling the algorithm when to use brute-force. ",1" forces it to use brute force immediately. Dunno what is preferable in our case?! 
```suggestion
        return self.waypoint_tree.query([x, y],1)[1]
```